content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__/list_clusters.rb

```diff
@@ -0,0 +1,45 @@
+module ManageIQ
+  module Automate
+    module Infrastructure
+      module VM
+        module Transform
+          module Import
+            class ListClusters
+              def initialize(handle = $evm)
+                @handle = handle
+              end
+
+              def main
+                values_hash = {}
+                values_hash[''] = '-- select cluster from list --'
+
+                provider_id = @handle.root['dialog_provider']
+                if !provider_id.nil? && !provider_id.empty? && provider_id != '!'
+                  provider = @handle.vmdb(:ext_management_system, provider_id)
+                  if provider.nil?
+                    values_hash['!'] = '-- No clusters found --'
+                  else
+                    provider.ems_clusters.each do |cluster|
+                      values_hash[cluster.id.to_s] = cluster.name
+                    end
+                  end
+                end
+                list_values = {
+                  'sort_by'   => :value,
```

Comment:
@matobet 
Do you want the Dialogs to show items sorted by ID or by description/name?
If you want it to displayed sorted by name description the sort_by => 'description'